target/linux/ramips/dts/mt7620a_edimax_br-6208ac-v2.dts

```diff
@@ -0,0 +1,257 @@
+			function = LED_FUNCTION_WLAN;
+			gpios = <&gpio2 3 GPIO_ACTIVE_LOW>;
+			linux,default-trigger = "phy0radio";
+		};
+
+		// RALINK_REG_PIO7140DATA 42-40
+		led_firmware: firmware {
+			label = "green:firmware";
+			color = <LED_COLOR_ID_GREEN>;
+			gpios = <&gpio2 2 GPIO_ACTIVE_LOW>;
+			// default-state = "keep";
+		};
+
+		// RALINK_REG_PIO7140DATA 41-40
+		led_vpn: vpn {
+			label = "green:vpn";
+			color = <LED_COLOR_ID_GREEN>;
+			gpios = <&gpio2 1 GPIO_ACTIVE_LOW>;
+		};
+	};
+
+	gpio_export {
+		compatible = "gpio-export";
+		#size-cells = <0>;
+		usb-power {
+			gpio-export,name="usb-power";
+			gpio-export,output=<1>;
+			gpios = <&gpio2 5 GPIO_ACTIVE_HIGH>;
+		};
+	};
+};
+
+&gdma {
+	status = "okay";
+};
+
+// Vendor firmware: RALINK_REG_PIO7140 - gpio0: gpio@600
+// Vendor firmware: RALINK_REG_PIO3924 - gpio1: gpio@638
+// Vendor firmware: RALINK_REG_PIO7140 - gpio2: gpio@660
+// Vendor firmware: RALINK_REG_PIO72   - gpio3: gpio@688
+
+&gpio1 {
+	status = "okay";
+};
+
+&gpio2 {
+	status = "okay";
+};
+
+&i2c {
+//	status = "okay";
+};
+
+&pcm {
+	status = "okay";
+};
+
+&spi0 {
+	status = "okay";
+
+	flash@0 {
+		compatible = "jedec,spi-nor";
+		reg = <0>;
+		spi-max-frequency = <10000000>;
+
+		partitions {
+			compatible = "fixed-partitions";
+			#address-cells = <1>;
+			#size-cells = <1>;
+
+			// Vendor partitions:
+			// 0x000000000000-0x000000030000 : "Bootloader"
+			// 0x000000030000-0x000000040000 : "Config"
+			// 0x000000040000-0x000000050000 : "Factory"
+			// 0x000000050000-0x000000070000 : "Cimage"
+			// 0x000000070000-0x000000fe0000 : "Uimage"
+			// 0x000000070000-0x000000190000 : "Kernel"
+			// 0x000000190000-0x000000fe0000 : "RootFS"
+			// 0x000000fe0000-0x000001000000 : "FreeSpace"
+
+			// Bootloader
+			partition@0 {
+				label = "u-boot";
+				reg = <0x0 0x30000>;
+				read-only;
+			};
+
+			// Config
+			partition@30000 {
+				label = "u-boot-env";
+				reg = <0x30000 0x10000>;
+				read-only;
+			};
+
+			// Factory
+			factory: partition@40000 {
+				label = "factory";
+				reg = <0x40000 0x10000>;
+				read-only;
+			};
+
+			// Cimage
+			partition@50000 {
+				label = "cimage";
+				reg = <0x50000 0x20000>;
+				read-only;
+			};
+
+			partition@70000 {
+				compatible = "openwrt,uimage", "denx,uimage";
+				openwrt,offset = <FW_EDIMAX_OFFSET>;
+				openwrt,partition-magic = <FW_MAGIC_EDIMAX>;
+				label = "firmware";
+				reg = <0x00070000 0x00f70000>;
+			};
+
+			// FreeSpace
+			partition@fe0000 {
+				label = "freespace";
+				reg = <0xfe0000 0x20000>;
+				read-only;
+			};
+		};
+	};
+};
+
+&state_default {
+	gpio {
+		// TODO: "spi refclk"?
+		groups = "i2c", "uartf", "rgmii1", "rgmii2", "ephy", "wled", "nd_sd";
+		function = "gpio";
+	};
+};
+
+&ethernet {
+	// pinctrl-names = "default";
+	// pinctrl-0 = <&ephy_pins>;
+	nvmem-cells = <&macaddr_factory_2e>;
+	nvmem-cell-names = "mac-address";
+
+	mediatek,portmap = "llllw";
+};
+
+&gsw {
+	mediatek,ephy-base = /bits/ 8 <12>;
+};
```

Comment:
I am not sure, would have to test.

Reply:
did you test the device without this line?

Reply:
No, I did not up to now because my first goal was running my router for at least one month. This goal is now achieved, so I can build new firmware and test that. What would go wrong if these lines were removed although required? I am not an expert for DTS files and therefore not sure what this part does.

Reply:
Please check if all network parts are still working without this setting. 
Just do some pings and check if nothing gets lost.

Reply:
After an upgrade with new firmware  the router is no longer accessible via ssh over ethernet. So it looks like these lines are required. I'll have to open the case and connect to the serial port to confirm that.